cpp/src/toppra/parametrizer/const_accel.hpp

```diff
@@ -16,6 +16,8 @@ class ConstAccel : public Parametrizer {
  public:
   ConstAccel(GeometricPathPtr path, const Vector &gridpoints, const Vector &vsquared);
 
+  const Vector& getTimes() const final { return m_ts; }
```

Comment:
```suggestion
  const Vector& getTimes() const override { return m_ts; }
```
Why `final` ? I think `override` is sufficient here. It is fine if a child class overrides this.